jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -47,12 +53,25 @@
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
   @Rule public TemporaryFolder outputFolder = new TemporaryFolder();
 
-  private CompilerUtil compilerUtil;
+  private CompilationTestHelper compilationTestHelper;
+
+  @BugPattern(
+      name = "DummyChecker",
+      summary = "Dummy checker to use CompilationTestHelper",
+      severity = WARNING)
+  /**
+   * A dummy checker to allow us to use {@link CompilationTestHelper} to compile Java code for
+   * testing, as it requires a {@link BugChecker} to run.
+   */
+  public static class DummyChecker extends BugChecker {
+    public DummyChecker() {}
+  }
 
   @Before
   public void setup() {
-    compilerUtil = new CompilerUtil(getClass());
-    compilerUtil.setArgs(Arrays.asList("-d", temporaryFolder.getRoot().getAbsolutePath()));
+    compilationTestHelper =
```

Comment:
This change was because `CompilerUtil` used a file manager class that got deleted from the EP test helpers.  This approach is also cleaner than what we had before

Reply:
Agreed. Makes sense to do this instead of duplicating code.